flake8_graphql.py

```diff
@@ -14,10 +16,13 @@ class GraphQLChecker(object):
     def __init__(self, tree, *args, **kwargs):
         self.tree = tree
 
-    messages = {
-        'GQL100': 'Cannot parse graphql syntax',
-    }
-
     def run(self):
         for node in ast.walk(self.tree):
-            print(node)
+            if isinstance(node, ast.Call) and hasattr(node.func, 'id') and node.func.id == 'gql':
+                if isinstance(node.args[0], ast.Str):
```

Comment:
you can collapse these two `if`s, `and` is lazy

also i find this style easier for long conditionals:

```py
if (
    foo and
    bar
):
```